create-near-test-accounts.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const { readKeyFile } = require('near-web3-provider/node_modules/near-api-js/lib/key_stores/unencrypted_file_system_keystore')
+
+async function setupNEARAccounts() {
```

Comment:
we really can't make this part of the setup? 
it's already called here: https://github.com/near/near-web3-provider/blob/master/src/index.js#L40

Reply:
To facilitate upstream acceptance, I think it'd be preferable to move all possible code to near-web3-provider--even if it wouldn't be used anywhere than by this repository. That also gives us more possibilities for changing or fixing things without bothering upstream again.

Reply:
So the reason why I landed on this ended up being a number of issues I ran into that felt like they exceeded a reasonable timebox.
I think the there are a couple of issues. We're trying to setup the accounts *while* truffle config is being loaded. Ultimately, the truffle config ends in `module.exports = {…` I tried adding async in a number of places, but that wasn't enough. Another issue is that constructors in JavaScript cannot be async. 
I tried the suggestions here:
https://stackoverflow.com/a/43433773/711863
but did not get this working. It would not reliably wait for all the accounts to be created.
So I think the constructor + the config file don't seem to be built this way. Then I thought about how it works with ganache-cli. Typically, ganache is started, then accounts are created, and then the tests are run with the truffle config. So I feel like Truffle is already rather separated from the account creation task.
I got to a point where I felt as though I was going to have to understand what Truffle wants and how it works and then said, "okay, I've spent enough time on this, just use a script."

Reply:
Added and merged here:
https://github.com/near/near-cli/pull/626